src/qibolab/serialize.py

```diff
@@ -202,6 +206,15 @@ def dump_qubit_name(name: QubitId) -> str:
     return name
 
 
+def load_component_config(
+    runcard: dict,
+    component: str,
+    config_class: type,
+) -> Config:
+    """Load configuration for given component."""
+    return config_class(**runcard["components"][component])
```

Comment:
Do we really need this? I understand that it's a helper from defining platforms, for example in https://github.com/qiboteam/qibolab/blob/982c843f41630f26a926af7ae2b38e720eb5f6dd/tests/dummy_qrc/zurich/platform.py#L63-L65
however we could even replace this with
```py
component_params = runcard["components"]
components[measure_lo] = OscillatorConfig(**component_params[measure_lo])
```
which is pure Python and does not need further familiarity of the user with `qibolab.serialize`.

This is more towards the direction of dropping serialize at some point, since if we make our objects simple enough we shouldn't need helpers.

Reply:
Sure